phpcs.xml

```diff
@@ -21,10 +23,31 @@
     <!-- PHP Rules -->
     <rule ref="PHPCompatibility" />
     <config name="testVersion" value="5.2-"/>
+    <rule ref="Squiz.PHP.DisallowMultipleAssignments.Found">
+        <severity>0</severity>
+    </rule>
 
     <!-- WordPress Rules -->
     <rule ref="WordPress">
         <exclude name="WordPress.VIP"/>
+        <!-- In the API, we process the requests. Nonce verification is to be done
+            at the form processing level prior to calling our API. -->
+        <exclude name="WordPress.CSRF.NonceVerification.NoNonceVerification"/>
+        <exclude name="WordPress.PHP.StrictInArray.MissingTrueStrict"/>
+        <!-- We have functions that can receive more arguments than declared
+            in the function's parameter list. -->
+        <exclude name="Squiz.Commenting.FunctionComment.ExtraParamComment"/>
+        <!-- Not every file needs a file header comment, e.g. Views -->
+        <exclude name="Squiz.Commenting.FileComment.Missing"/>
+        <!-- There are valid areas requiring errors to be silenced. -->
+        <exclude name="Generic.PHP.NoSilencedErrors.Discouraged"/>
```

Comment:
As of PHPCS 3.2.0, you can [selectively ignore specific sniffs on specific lines](https://github.com/squizlabs/PHP_CodeSniffer/blob/8a18d4049dda4349cfe01dffc1420bbce31d304e/package.xml#L47), so this may be a better approach, rather than disabling the whole check, which means that other silenced errors may be accidentally introduced.

Reply:
@GaryJones With this new change in 3.2.0, is this done within the `phpcs.xml` file or as an annotation?

Reply:
Per line comment. i.e.:
```php
@ini_set .... // phpcs:disable Generic.PHP.NoSilencedErrors.Discouraged
```
(or something similar)

Reply:
@GaryJones Implemented.  But upon running we are getting Warnings:

```
 90 | WARNING | Silencing errors is discouraged (Generic.PHP.NoSilencedErrors.Discouraged)
 94 | WARNING | Silencing errors is discouraged (Generic.PHP.NoSilencedErrors.Discouraged)
```